modules/nw-overlapped-sharding.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-cluster-traffic-ingress-controller.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-overlapped-sharding_{context}"]
+== Overlapped sharding example
+
+In addition to `finops-router` and `dev-router` in the example above, you also have `devops-router`, which is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `dev` and `ops`:
```

Comment:
```suggestion
In addition to `finops-router` and `dev-router` in the previous example, you also have `devops-router`, which is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `dev` and `ops`:
```
I know this is copied & pasted text but maybe it could be changed while you are in the files?